Files/Local Files/Copy a Single file from Fileshare to SFTP/process.json

```diff
@@ -0,0 +1,67 @@
+﻿{
```

Comment:
Inside the process for all process variables - can you change the word "move" to "copy". Eg. "Name of file to be moved" should IMO be "Name of file to be copied", since moving something usually implies that the file will not be there after the operation. But in this template the source file stays where it originally was.